janus_core/cli/phonons.py

```diff
@@ -29,8 +29,19 @@ def phonons(
     ctx: Context,
     struct: StructPath,
     supercell: Annotated[
-        tuple[int, int, int], Option(help="Supercell lattice vectors.")
-    ] = (2, 2, 2),
+        str,
+        Option(
+            help="Supercell specification as a string, in the Phonopy style. "
+            "There are three different valid inputs: single value (i.e. '2') "
+            "which specifies what all of the diagonal values of the supercell "
+            "matrix whould be, three values (i.e. '1 2 3') which specify the "
+            "individual diagonal elements of the supercell matrix, or nine "
+            "values (i.e. '1 2 3 4 5 6 7 8 9') which specify all the elements "
+            "of the full supercell matrix. In the last case, the first three "
+            "values form the first row of the matrix, the second three the "
+            "second row, etc."
```

Comment:
```suggestion
            help="Supercell matrix, in the Phonopy style. Must be passed as a string "
            "in one of three forms: single integer ('2'), which specifies all "
            "diagonal elements; three integers ('1 2 3'), which specifies each "
            "individual diagonal element; or nine values ('1 2 3 4 5 6 7 8 9'), which "
            "specifies all elements, filling the matrix row-wise."
```

Maybe, to be slightly more concise for the CLI help in particular? If not, then just note there's a typo: `whould` here and in the `supercell : str` description

Reply:
Adam was ok with these rewrites, so I have added them as is, thanks!